src/FlyCube/CommandList/DXCommandList.cpp

```diff
@@ -732,6 +732,16 @@ void DXCommandList::ResolveQueryData(const std::shared_ptr<QueryHeap>& query_hea
                                                  D3D12_RESOURCE_STATE_UNORDERED_ACCESS, 0));
 }
 
+void DXCommandList::SetGraphicsConstant(uint32_t root_parameter_index, uint32_t value, uint32_t byte_offset)
```

Comment:
In the current implementation, how do you know which `root_parameter_index` value you need to pass?

Reply:
The root parameter index is the index of the binding set/layout set in the list given to CreateBindingSetLayout or CreateBindingSet.

It's not very practical as you need to keep track of the index in the binding list of the constant buffer you want to update. Maybe we could change the prototype to use a BindKey instead?